src/util/serviceSelector.ts

```diff
@@ -0,0 +1,106 @@
+import {  ProcessorRequest, RouterRequestData } from "../types/index";
+import { MiscService } from "../services/misc.service";
+import { INTEGRATION_SERVICE } from "../routes/utils/constants";
+import CDKV1ServiceDestination from "../services/destination/cdkV1Integration.destination.service";
+import CDKV2ServiceDestination from "../services/destination/cdkV2Integration.destination.service";
+import IntegrationServiceDestination from "../interfaces/IntegrationDestinationService";
+import NativeIntegrationServiceDestination from "../services/destination/nativentegration.destination.service";
+import IntegrationServiceSource from "../interfaces/IntegrationSourceService";
+import NativeIntegrationServiceSource from "../services/source/nativeIntegration.source.service";
+
+export class ServiceSelector {
+  private static sourceHandlerMap: Map<string, any> = new Map();
+  private static destHandlerMap: Map<string, any> = new Map();
+  private static serviceMap: Map<string, any> = new Map();
```

Comment:
How about a multi-level map to hold all these together?

Reply:
It improves extensibility. Also, aligns with the tags structure we are now coming up with.

Reply:
I have relocated these into respective classes but I have kept sourceHandler and destinationHandler different as there is not major overlap on native handlers